source/change-streams/tests/change-streams-errors.json

```diff
@@ -102,10 +102,7 @@
       ],
       "result": {
         "error": {
-          "code": 280,
-          "errorLabels": [
-            "NonResumableChangeStreamError"
-          ]
+          "code": 20
```

Comment:
As part of the changes for SERVER-45505, the query team changed this from generic error code 280 (`ChangeStreamFatalError`) to more specific error code 20 (`IllegalOperation`). I left this test in because it asserts that errors not in the whitelist are propagated to the user.

Reply:
And cause problems for older drivers?  At the least, test expectations will be different.